pkg/operator/operator/k8s.go

```diff
@@ -971,6 +1006,28 @@ func getEnvVars(api *spec.API, container string) []kcore.EnvVar {
 	return envVars
 }
 
+func getKubexitEnvVars(containerName string, deathDeps ...string) []kcore.EnvVar {
+	envVars := []kcore.EnvVar{
+		{
+			Name:  "KUBEXIT_NAME",
+			Value: containerName,
+		},
+		{
+			Name:  "KUBEXIT_GRAVEYARD",
+			Value: _kubexitGraveyardMountPath,
+		},
+	}
+
+	if deathDeps != nil {
+		envVars = append(envVars, kcore.EnvVar{
+			Name:  "KUBEXIT_DEATH_DEPS",
```

Comment:
robert: what is this one for?

Reply:
for defining the death dependencies (e.g. tf serving has death dependency on the API container)